sandbox/src/LargePayload.js

```diff
@@ -8,7 +8,7 @@ export default function LargePayload() {
       const payload = new Uint8Array(size * 1024);
       window.alloy("event", {
         documentUnloading: true,
-        data: {
+        xdm: {
           payload
```

Comment:
This isn't xdm.

Reply:
Thanks @jonsnyder. I've switched it back to `data`.